### PR TITLE
updated breeze (1.5.11)

### PR DESCRIPTION
**Pull request by vitorgalvao**

### CODE REVIEW

**Comment by adityadalal924** [comment]
Merged as e1a0eee733c51b499374fe47ed02dd75e20846be.

File: Casks/breeze.rb [hunk 1]
```diff
@@ -3,8 +3,6 @@
   sha256 'eaa54f9ccc6539fb548ffa274a7317913840ee3144ed781acd4da8b0b2d0d5ba'
 
   url "https://autumnapps.com/downloads/Breeze#{version}.dmg"
-  appcast 'https://www.autumnapps.com/updates/profiles/breeze.php',
```
Comment (tsparber) [inline Casks/breeze.rb]: Maybe the mysql error is only temporary...
(So removing the appcast might loose the ability for automatic checks)

Comment (vitorgalvao) [inline Casks/breeze.rb]: It has been there for a while (either that or I always bump against it when it is down).

Comment (tsparber) [inline Casks/breeze.rb]: Ok :smiley:

**Pull request closed without merging**